dependencies.gradle

```diff
@@ -8,10 +8,23 @@ dependencies {
     compile("com.github.GTNewHorizons:Avaritia:1.23:dev")
     compile("net.industrial-craft:industrialcraft-2:2.2.828-experimental:dev")
     compile("com.github.GTNewHorizons:GTplusplus:1.7.62-pre:dev")
+    compile("com.github.GTNewHorizons:AE2FluidCraft-Rework:1.0.6-gtnh:dev")
 
     compileOnly("com.github.GTNewHorizons:Hardcore-Ender-Expansion:1.9.1-GTNH:dev") {
         transitive = false
     }
+    compileOnly("com.github.GTNewHorizons:AE2FluidCraft-Rework:1.0.6-gtnh:dev") {
```

Comment:
why adding it two times? also if it's just added so you can see the recipe in dev, that is not really required: the script if done correctly will load itself only if all the mods involved in the recipe are present. So you can put an AE2FC dev jar in your dev env's mods folder instead.